modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/resources/META-INF/search/mappings.json

```diff
@@ -1731,6 +1731,12 @@
 				"type": "text"
 			},
 			"assetType": {
+				"fields": {
+					"keyword": {
+						"store": true,
```

Comment:
I couldn't find assetType being indexed anywhere, are we sure this is actually used?

Reply:
Hi @joshuacords ,

I think you are correct, the assetType field isn't being indexed, but instead it's the assetType_\<languageId> and assetType_\<languageId>_sortable fields that are being indexed.

And so, we would actually need to instead create the keyword sub-field for those dynamic templates instead, like we did for the liferay-type-mappings:
- Reference: 
  - [mappings.json#L6-L414](https://github.com/liferay/liferay-portal/blob/7.4.3.80-ga80/modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/resources/META-INF/search/mappings.json#L6-L414)
  - [mappings.json#L1312-L1721](https://github.com/liferay/liferay-portal/blob/7.4.3.80-ga80/modules/dxp/apps/portal-workflow/portal-workflow-metrics-service/src/main/resources/META-INF/search/mappings.json#L1312-L1721)